src/Tests/Microsoft.TemplateEngine.Cli.UnitTests/Microsoft.TemplateEngine.Cli.UnitTests.csproj

```diff
@@ -15,9 +15,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="FakeItEasy" />
-    <PackageReference Include="Verify.XUnit" />
-    <PackageReference Include="Verify.DiffPlex" />
+    <PackageReference Include="FakeItEasy" Version="7.3.1" />
+    <PackageReference Include="Moq" Version="4.8.2" />
```

Comment:
@vlada-shubina , I suspect it can be the undesired thing to add a new testing package, can't it?

Reply:
Both are already used. Please use `$(MoqPackageVersion)` instead of version (it is defined here https://github.com/dotnet/sdk/blob/8546a2ea939238c0dbf7285f0a863794af73534f/eng/Versions.props#L191) and keep `FakeItEasy` without version (it is managed by dependabot). 
```suggestion
    <PackageReference Include="Moq" Version="$(MoqPackageVersion)" />
    <PackageReference Include="FakeItEasy" />
```

Reply:
Done.